client/packages/system/src/Manage/Preferences/EditPage/EditPreference.tsx

```diff
@@ -73,7 +73,13 @@ export const EditPreference = ({
       if (!isNumber(preference.value)) {
         return t('error.something-wrong');
       }
-      return <NumericTextInput value={value} onChange={handleChange} />;
+      return (
+        <NumericTextInput
+          value={value}
+          onChange={handleChange}
+          onBlur={() => {}}
```

Comment:
`onBlur` is causing the modal to call the update twice. `onChange` is more than enough for update. Having `onBlur` introduced some issues as well for disabling the Ok Button for the modal because it gets called twice so the state resets on the second call